oauth2/fosite_store_sql.go

```diff
@@ -200,6 +211,7 @@ func (s *FositeSQLStore) hashSignature(signature, table string) string {
 }
 
 func (s *FositeSQLStore) createSession(ctx context.Context, signature string, requester fosite.Requester, table string) error {
+	dbAccess := s.getDbAccess(ctx)
```

Comment:
dbAccess is both a type and a value here. Maybe just rename this to `db := s.db()`?